### PR TITLE
added diagrams

### PR DESCRIPTION
**Pull request by ilariae**
### Description

Added diagrams for gateway and Lifetime of a message VAA

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [ ] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

File: learn/messaging/gateway.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The following flow outlines the key steps involved in a cross-chain transaction
 5. **Finalization on target Cosmos chain** - the VAA is verified on the target Cosmos chain, enabling the execution of the corresponding transaction or operation
 6. **Global Accountant integration** - throughout the process, the Gateway works with the Global Accountant to maintain accurate asset tracking, ensuring the integrity of cross-chain transactions
 
-<!-- add diagram here -->
+![Gateway flow diagram](/docs/images/learn/messaging/gateway/gateway-diagram.webp)
```
Comment (eshaben) [inline learn/messaging/gateway.md:27]: Nice to see this come to life! Sorry I dropped the ball on getting this one over to the designer a while ago 😅 

This is just a nitpick, but under Source Contract, "initiates" isn't capitalized. Similarly, under Final Transaction, "is" isn't capitalized. But for all the other steps they are capitalized 🤷‍♀️  - can we easily fix this? Or does the designer have to do it?

Comment (ilariae) [reply learn/messaging/gateway.md:27]: the designers would have to do it I cant modify it myself

Comment (ilariae) [reply learn/messaging/gateway.md:27]: I'll ask them

**Comment by ilariae** [comment]
updated gateway diagram

**Review by eshaben — APPROVED** [review]
Beautiful! Thanks!

**Pull request merged**